### PR TITLE
refactor: use entry.unique_id in entity unique_id

### PR DESCRIPTION
**Pull request by vinteo**

### CODE REVIEW

File: custom_components/opensprinkler/__init__.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def _get_station_attributes(self, station):
     def device_info(self):
         """Return device information about Opensprinkler Controller."""
         return {
-            "identifiers": {(DOMAIN, self._entry_id)},
+            "identifiers": {(DOMAIN, self._entry.unique_id)},
```
Comment (vinteo) [inline custom_components/opensprinkler/__init__.py:225]: do I need to slugify here?

Comment (vinteo) [reply custom_components/opensprinkler/__init__.py]: added it

**Comment by travisghansen** [comment]
@vinteo should we also `slugify` the value here? `await self.async_set_unique_id(user_input[CONF_MAC])`

**Comment by vinteo** [comment]
maybe it would be a good idea, i think it will convert to lower case as well so it would account for user variation in that

**Comment by travisghansen** [comment]
Yeah, let's do it. Then I'll merge.

**Comment by vinteo** [comment]
done

**Pull request merged**